index.html

```diff
@@ -1355,6 +1367,199 @@ <h4>Exchange Examples</h4>
       </section>
     </section>
 
+    <section>
+      <h3>Initiating Interactions</h3>
+      <p>
+It is useful for an implementation to communicate how to start interacting with
+it to another implementation. This bootstrapping process is called
+<em>initiating an <dfn>interaction</dfn></em>, and communicates what protocols
+each implementation supports as well as how to start a particular interaction
+with the implementation.
+      </p>
+      <p>
+A high-level overview of a Holder initiating an interaction using a QR Code and
+a VC API Exchange is shown below:
+      </p>
+      <figure>
+        <pre class="mermaid">
+sequenceDiagram
+  participant H as Holder
+  participant W as Holder Coordinator (Wallet)
+  participant V as Verifier Coordinator
+  autonumber
+  H->>W: Tap "Share credential"
+  W->>W: Generate Interaction URL
+  W->>W: Display QR Code
+  H->>V: Present QR Code
+  V->>V: Scan QR Code and decode URL
+  V->>W: Get available interaction protocols
+  Note left of V: GET &lt;https://wallet.example/interactions/&lt;interactionId>?wi_api=&lt;api_version>
+  W->>V: Return interaction protocols
+  Note left of V: (VC API, OID4, NFC, Bluetooth, etc.)
+  V->>V: Select appropriate protocol
+  V->>W: Transmit protocol selection and options
+  W->>H: Request consent to proceed
+  Note right of H: "Open browser to verifier.example?"
+  H->>W: Confirm consent start exchange
+  W->>V: Start exchange
+        </pre>
+        <figcaption>
+A Holder-initiated exchange using a QR Code
+        </figcaption>
+      </figure>
+
+      <p class="note" title="Multiple interaction initiation variations are possible">
+While the example above is a [=holder=]-initiated interaction, [=issuers=] and
+[=verifiers=] are also capable of initiating an interaction. Similarly, while
+the transmission of the interaction URL here is performed via a QR Code
+(optical medium), an interaction URL can also be transmitted over NFC or
+Bluetooth (wireless medium).
+      </p>
+
+      <section>
+        <h4>Interaction URL Format</h4>
+        <p>
+The format of the <dfn>interaction URL</dfn> MUST conform to the syntax for the
+[[[URL]]] and contain an `iuv` query parameter encoding the interaction URL
+version number, which MUST be `1`. The interaction URL SHOULD be an HTTPS URL
```

Comment:
The rationale for HTTPS should be explained here -- which includes all of the trust signals and benefits with origins -- and how delegation to other service providers can happen simply by retrieving the protocols object via TLS from the trusted origin (over HTTPS). This helps trust, it helps prevent vendor lock, and it does it simply. Any other URL scheme that might be considered should know about this / maybe make a statement about whether or not these same features are supported.

Reply:
Fixed in e37e305ea346a04d67b6a5fe5805ed54b0c7ccb2.